src/components/WalletUnlock/Hardware.tsx

```diff
@@ -2,17 +2,8 @@ import React from 'react';
 
 import styled from 'styled-components';
 
-import {
-  Box,
-  BusyBottom,
-  Button,
-  Heading,
-  Icon,
-  InlineMessage,
-  Spinner,
-  Text,
-  TIcon
-} from '@components';
+import { Box, BusyBottom, Button, Heading, InlineMessage, Spinner, Text, TIcon } from '@components';
+import Icon from '@components/Icon';
```

Comment:
Import with the others

Reply:
Was doing this previously, but i think this was leading to a circular dependency in storybook builds